pkgs/tools/system/runitor/default.nix

```
@@ -0,0 +1,43 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, runitor }:
```

You shouldn't need to use `testers` here as `testVersion` is already exposed, I think:
```suggestion
{ lib, buildGoModule, fetchFromGitHub, testVersion, runitor }:
```

---

I think the preferred way is to use `testers.testVersion` now.

There was a tree-wide search-and-replace recently: f1c7f19e49190cf58041908f9c194431e2513688

...and these are the current stats:
```
nixpkgs% rg -l '= testers.testVersion' pkgs | wc -l
62
nixpkgs% rg -l '= testVersion' pkgs | wc -l
5
```